pkg/os/device.go

```diff
@@ -96,3 +102,29 @@ func IsExists(f string) bool {
 	_, err := os.Stat(f)
 	return !os.IsNotExist(err)
 }
+
+// ParseSize parses disk size string (e.g. "10GB" or "150MB") into MegaBytes
+// If sizeStr contains number without unit (e.g. "150"), then MB is assumed.
+func ParseSize(sizeStr string) (MegaBytes, error) {
+	// assume MB if no unit is passed
+	rNoUnit, _ := regexp.Compile("([0-9]+)$")
```

Comment:
I suggest to change the pattern to `"^[0-9]+$"`. Two changes:

 * use of `^` at the beginning will require that the text starts with a number (it only contains a number)
 * there is no selector in the regex `()` - you don't actually need to capture any data at this point. This also requires that you change the check in the next line as in case of a match, you will get just a single element, not two.

I see that other regexes will also allow for such texts which I think should be fixed.